### PR TITLE
AtCore usage improvements

### PR DESCRIPTION
**Pull request by sithlord48**
Fixed the issues preventing atcore from sending correct data..
added a print / resume on pause clicked . 
Emergency stop will send m112 if the printer is not busy

### CODE REVIEW

File: src/atelier.cpp [hunk 1]
```diff
@@ -39,6 +39,9 @@ void Atelier::checkPrinterStatus(PrinterStatus status)
 
 void Atelier::pausePrint()
 {
+    if (core->state() ==PrinterState::PAUSE) {
```
Comment (patrickelectric) [inline src/atelier.cpp]: Space == space

File: src/atelier.cpp [hunk 2]
```diff
@@ -49,7 +52,11 @@ void Atelier::stopPrint()
 
 void Atelier::emergencyStopPrint()
 {
-    core->setState(PrinterState::STOP);
+    if (core->state() == PrinterState::BUSY) {
+        core->setState(PrinterState::STOP);
+    } else {
+        core->pushCommand("M112");
```
Comment (patrickelectric) [inline src/atelier.cpp]: Use atcore

File: src/atelier.cpp [hunk 3]
```diff
@@ -72,35 +79,34 @@ void Atelier::homeZ()
...
-        core->move(AXIS::Y, dist.toUInt());
+    if (axis == QLatin1Char('X'))
+        core->move(AXIS::X, dist);
+    else if (axis == QLatin1Char('Y'))
+        core->move(AXIS::Y, dist);
     else
-        core->move(AXIS::Z, dist.toUInt());
-
+        core->move(AXIS::Z, dist);
 }
```
Comment (patrickelectric) [inline src/atelier.cpp]: Int axis =( axis == 'X')&&axis::X | (axis=='Y')&&axis::Y&&andthesameforZ
Core move (axis)

Comment (sithlord48) [inline src/atelier.cpp]: don't have time to test right now. will change when i can test.

**Pull request merged**